### PR TITLE
Prettify error messages of QOM tree updater

### PR DESCRIPTION
**Pull request by laerreal**
Some code has been moved to make it reusable.

### v3
- remove constant condition
- coding style

### v2
- remove blank lines between traceback lines

### CODE REVIEW

File: common/co_dispatcher.py [hunk 1]
```diff
@@ -71,6 +73,35 @@ def __init__(self,
...
+        task = self
+        e = task.exception
+
+        if e is None:
+            return []
+
+        lines = []
+
+        while isinstance(e, (CancelledCallee, FailedCallee)):
+            if task is not None:
```
Comment (BDanAnd) [inline common/co_dispatcher.py]: Can there be a situation when the `e.callee` is `None`?

File: qemu/version_description.py [hunk 2]
```diff
@@ -806,6 +808,10 @@ def co_init_device_tree(self, targets = None):
                     arch,
                     root
                 )
+            except (CancelledCallee, FailedCallee) as e:
+                message = e.callee.traceback_lines
+                message.insert(0, "Device Tree for %s isn't created:" % arch)
+                print("\n".join(message))
```
Comment (BDanAnd) [inline qemu/version_description.py]: Maybe it’s better to do like this?
```
message.insert(0, "Device Tree for %s isn't created:\n" % arch)
print("".join(message))
```

**Review by BDanAnd — COMMENTED** [review]
v1 is viewed

File: common/co_dispatcher.py (hunk 1 above)
Comment (laerreal) [reply common/co_dispatcher.py]: Normally not. If so, it's an error somewhere else.

File: qemu/version_description.py (hunk 2 above)
Comment (laerreal) [reply qemu/version_description.py]: oops...

**Comment by laerreal** [comment]
v2 is ready

**Comment by laerreal** [comment]
v3 is ready

**Pull request merged**